tests/test_shipping.py

```diff
@@ -16,11 +16,19 @@ def test_upsert_dewar(testconfig):
         params['name'] = 'Test-dewar'
         params['comments'] = 'This is a test ...'
         #params['barcode'] = 'cm1-1-i03-0023151' # must be unique! Or not set ...
-        params['status'] = 'at DLS'
+        params['status'] = 'at facility'
         params['type'] = 'Dewar' # only Dewar and Toolbox allowed in table definition
-        sid = shipping.upsert_dewar(list(params.values()))
-        assert sid is not None
-        assert sid > 0
+        did = shipping.upsert_dewar(list(params.values()))
+        assert did is not None
+        assert did > 0
```

Comment:
Generic remark:
```python
assert x is not None
```
is implied by
```python
assert x > 0
```
so the first line is not _strictly_ required.